src/modules/lreproxy/lreproxy.c

```diff
@@ -0,0 +1,1976 @@
+/*
+ * Copyright (C) 2019-2020 Mojtaba Esfandiari.S, Nasim-Telecom
```

Comment:
From brief look to it, this is pretty much carboon-copy of the rtpproxy module, as such original (c) line of the code should be reproduced. Thanks!

Reply:
For cause of compatibilities with other codes in Kamailio, I decided to use rtpproxy module,such as naming of functions, parsing URL, SDP, get media descriptions and etc. But the process of relaying media is completely different. The LREProxy uses PREROUTING in netfilter hooks in Linux networking stack.
The LREProxy engine is running pass the last steps of developing, Because it needs to use netlink library for interacting between user-pace and kernel-space, it took more time.